ci-operator/config/securesign/cosign/securesign-cosign-release-next.yaml

```diff
@@ -0,0 +1,23 @@
+binary_build_commands: make cosign
+build_root:
+  image_stream_tag:
+    name: release
```

Comment:
We don't have `golang-1.20` yet:
```sh
$ oc --context=app.ci -n openshift get istag/release:golang-1.20
Error from server (NotFound): imagestreamtags.image.openshift.io "release:golang-1.20" not found
```

Reply:
@JasonPowr I believe you should be able to build with Go 1.20 using this image stream tag.

https://github.com/openshift/release/blob/1bd82837937cf107e83a32900496f3c61b5ccc2f/ci-operator/config/securesign/rekor/securesign-rekor-redhat-v1.2.2.yaml#L2-L5